README.md

```diff
@@ -81,6 +81,10 @@ RIOT provides features including, but not limited to:
 * LoRaWAN
 * UWB
 * Bluetooth (BLE) via [NimBLE](https://github.com/apache/mynewt-nimble)
+* Rust support
```

Comment:
I think just `Rust` would be misleading, as readers may assume RIOT would be written primarily/purely in Rust.

Probably it would make sense to add some more structure. E.g.

 * Platforms Supported
    - 200+ boards based on AVR, MSP430, ESP8266, ESP32, RISC-V, ARM7, and ARM Cortex-M
    - the native port allows to run RIOT as-is on Linux and BSD.
      Multiple instances of RIOT running on a single machine can also be
      interconnected via a simple virtual Ethernet bridge or via a simulated
      IEEE 802.15.4 network (ZEP)
 * Programming Environment
    - Write native code in C, C++, or Rust
    - Rely upon POSIX APIs, user friendly hardware abstractions, and standard embedded Rust interfaces
    - Integrate dynamic runtimes such as MicroPython, elm (JavaScript), and WASM
 * Network Features and Protocols
    - IPv6
    - 6LoWPAN (RFC4944, RFC6282, and RFC6775)
    - UDP
    - RPL (storing mode, P2P mode)
    - CoAP
    - MQTT
    - Bluetooth (BLE) via [NimBLE](https://github.com/apache/mynewt-nimble)
    - LoRaWAN
    - UWB
    - CNN-Lite
 * System Features
    - a preemptive, tickless scheduler with priorities
    - flexible memory management
    - high resolution, long-term timers
    - MTD abstraction layer for memory devices
 * Security Features
    - OTA updates via SUIT
    - PSA Cryptographic API
    - DTLS, EDHOC, ...
    - ...
 * Device Drivers
    - SPI, I2C, UART, USB, ...
    - Various sensors: Environmental sensors, IMUs, magnetometers, particulate matter sensors, ...
    - Various actuators: Servos, motors, dimmers, switches, RGB LEDs, ...
    - Displays and touchscreens
    - GNSS modules
    - ...

But as said, I'd rather have this discussion split out.